src/matrixroomhandler.ts

```diff
@@ -271,8 +271,25 @@ export class MatrixRoomHandler {
                 });
             }
 
-            const guildId = args[0];
-            const channelId = args[1];
+            let guildId: string;
+            let channelId: string;
+
+            const AMOUNT_OF_IDS_DISCORD_IDENTIFIES_ROOMS_BY = 2;
+
+            if (args.length === AMOUNT_OF_IDS_DISCORD_IDENTIFIES_ROOMS_BY) { // "x y" syntax
+                guildId = args[0];
+                channelId = args[1];
+            } else if (args.length === 1 && args[0].includes("/")) { // "x/y" syntax
+                const split = args[0].split("/");
+                guildId = split[0];
+                channelId = split[1];
+            } else {
+                return this.bridge.getIntent().sendMessage(event.room_id, {
+                    formatted_body: "Invalid syntax: See <code>!discord help</code>",
```

Comment:
you still need a `body` even if you have a `formatted_body`, else that is invalid matrix stuff

Reply:
`formated_body` is basically just there to make things look pretty in clients that support it, and `body` is the fallback that always has to be htere

Reply:
So `body` should be "Invalid syntax: See !discord help"?

Reply:
preferably with backticks:
```
"Invalid syntax: See `!discord help`"
```

Reply:
Ah, so markdown?

Reply:
yeah, that seems to be the thing most use as fallback